src/api/projects/projects.service.ts

```diff
@@ -65,6 +65,8 @@ export class ProjectsService {
       delete project.mpAccessToken;
       delete project.mpPublicKey;
       delete project.mpInstantCheckout;
+      delete project.moneyEarned;
```

Comment:
don't we want to show the money earnet in the FE?

Reply:
For the Admins yes, like in the dashboard